shared/constants/settings.js

```diff
@@ -35,6 +35,9 @@ const initialState: Types.State = {
     newPassphraseError: null,
     rememberPassphrase: true,
   },
+  chat: {
```

Comment:
Looks like `lockdownModeEnabled` already messed this up, but object keys are supposed to be alphabetical.  (eslint warns about it but doesn't fix it.)